Lib/test/test_asyncio/test_futures2.py

```diff
@@ -0,0 +1,15 @@
+# IsolatedAsyncioTestCase based tests
```

Comment:
Is the motivation here to start a new set of tests using `IsolatedAsyncioTestCase` for simplification? If so, is there going to be some effort to gradually migrate the existing `test_futures` or is it just for new ones?

Reply:
Yes, the gradual migration of the asyncio test suite would be nice.
I believe `IsolatedAsyncioTestCase` based tests are much more readable.
On another hand, don't fix if not broken :)  Plus tests rewriting takes time with very little effort.
For a new test, I prefer the new approach, definitely.